### ISSUE TITLE
script loading error/ client crash on reading book entry

### ISSUE THREAD

**Issue by IRSmoh** [Yuudaari/soulus#65]
the first block (separated by a large amount of white space) is related to the script failing to load. the second half is a client crash that occurs when attempting to read the first entry in the book.

https://gist.github.com/IRSmoh/eb72636912a323b2b6bbc0c0c7dcc5d3

ver: Soulus-1.12.2-4.0.2-b18da36

**Comment by Yuudaari** [comment]
Could I have your Patchouli & Ender IO versions as well?

**Comment by IRSmoh** [comment]
Patchouli-1.0-16

EnderIO-base-1.12.2-5.0.40
EnderIO-conduits-1.12.2-5.0.40
EnderIO-conduits-appliedenergistics-1.12.2-5.0.40
EnderIO-endergy-1.12.2-5.0.40
EnderIO-integration-tic-1.12.2-5.0.40
EnderIO-machines-1.12.2-5.0.40

those are all the enderio addons we have

**Comment by Yuudaari** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Patchouli/Ender IO crash](https://trello.com/c/VwYYcpSD/293-patchouli-ender-io-crash)

**Comment by Yuudaari** [comment]
Hmm, it looks like I can't reproduce with just these mods. I'm able to read entries in the Soulus guidebook with no problems, and I don't have any errors in the log. Unfortunately I can't go through and test a whole bunch of versions of mods until finding one that crashes (I don't have the time), so I can't follow up on this until I have a way to reproduce the issue consistently.

If you remove all the mods in your pack until all that's left is EnderIO, Patchouli, Soulus, and JEI, do you still get the issue? I think it's more likely to be issues in EnderIO and Patchouli than in Soulus.

**Comment by IRSmoh** [comment]
Honestly, no idea we're able to fairly reliably run into it on the FTB interactions beta (curr ver: https://drive.google.com/file/d/19zkTLcG5iUrzpNog8AWWALYaLDS9-BYM/view)

tldr blame vyraal for all problems

**Comment by Yuudaari** [comment]
https://github.com/Vazkii/Patchouli/issues/110

Relevant Patchouli issue
